app/gui/src/dashboard/pages/dashboard/components/Label.tsx

```diff
@@ -87,6 +87,7 @@ export default forwardRef(function Label(
 
           {onDelete && (
             <Button
+              href="#"
```

Comment:
Why this is needed?

Reply:
Yes, this code is going off-rails. This case should be clearly documented. And let links be links and buttons be buttons.

Also, even with links inside buttons, still there is confusion which one is actually clicked.